src/datatypes/aktivitetTypes.ts

```diff
@@ -205,5 +205,5 @@ export enum ForhaandsorienteringType {
 export interface Forhaandsorientering {
     type: ForhaandsorienteringType;
     tekst: string;
-    lestDato?: string;
```

Comment:
føler det er veldig greit at den heter lestDato, siden det er en string

Reply:
Egentlig enig, men hva heter den i databasen? Den var endret til `lest` en plass som gjorde at mye av logikken ble feil for strenge beskjeder 🤔

Reply:
I database tror jeg den heter lest. Men der er den typet til date.

Reply:
Da må vi evt mappe den om til lestDato etter at vi får dataen

Reply:
tenker vi gjør endringen du het gjort. Mapper nesten ikke om noe uansett